Mathlib/Algebra/AddTorsor.lean

```diff
@@ -467,20 +467,20 @@ theorem pointReflection_involutive (x : P) : Involutive (pointReflection x : P 
 set_option linter.deprecated false
```

Comment:
I'm guessing all of those can go now

Reply:
I would like to keep this diff as small as possible. And I also think that many of these lemmas can be removed or renamed in a second pass.